pkg/version/version.go

```diff
@@ -73,31 +74,16 @@ func DeployVersion(appID string, sequence int64) error {
 		}
 	}
 
-	db := persistence.MustGetDBSession()
-
 	logger.Info("deploying app version", zap.String("appId", appID), zap.Int64("sequence", sequence))
 
-	tx, err := db.Begin()
-	if err != nil {
-		return errors.Wrap(err, "failed to begin")
-	}
-	defer tx.Rollback()
-
+	db := persistence.MustGetDBSession()
 	query := `update app_downstream set current_sequence = $1 where app_id = $2`
-	_, err = tx.Exec(query, sequence, appID)
+	_, err = db.Exec(query, sequence, appID)
 	if err != nil {
 		return errors.Wrap(err, "failed to update app downstream current sequence")
 	}
 
-	query = `update app_downstream_version set status = $3, applied_at = $4 where sequence = $1 and app_id = $2`
-	_, err = tx.Exec(query, sequence, appID, storetypes.VersionDeploying, time.Now())
-	if err != nil {
-		return errors.Wrap(err, "failed to update app downstream version status")
-	}
-
-	if err := tx.Commit(); err != nil {
-		return errors.Wrap(err, "failed to commit")
-	}
+	go operator.DeployApp(appID, sequence)
```

Comment:
I think this requires `DeployApp` to be thread safe

Reply:
Could also keep the loop and make it read from a channel so only one deployment happens at a time.

Also if kotsadm restarts before deployment happens, someone will have to click the button again.

Reply:
`DeployApp` should be thread safe since it uses a mutex? but i do agree with your other point. i'll close this PR then for now.

Reply:
also, the loop already does one deployment at a time.